Source/Modules/ModuleInput.cpp

```diff
@@ -17,6 +17,11 @@
 	#include "optick.h"
 #endif // DEBUG
 
+namespace
+{
+	const float COOLDOWN_SECONDS = 0.25f;
+} // namespace
+
 ModuleInput::ModuleInput() : mouseWheel(float2::zero), mouseMotion(float2::zero), mousePosX(0), mousePosY(0)
```

Comment:
Regarding the initializer listst comment, you might be able to do this
```suggestion
ModuleInput::ModuleInput() : mouseWheel(float2::zero), mouseMotion(float2::zero), mousePosX(0), mousePosY(0), keysCooldown{ COOLDOWN_SECONDS }
```
Same for the rest.

Reply:
done!